python/datasources/cdc_hiv_black_women.py

```diff
@@ -0,0 +1,116 @@
+import pandas as pd
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util, standardized_columns as std_col
+from ingestion.constants import NATIONAL_LEVEL, US_FIPS, CURRENT, HISTORICAL
+from ingestion.het_types import GEO_TYPE, HIV_BREAKDOWN_TYPE
+from ingestion.cdc_hiv_utils import (
+    load_atlas_df_from_data_dir,
+    BREAKDOWN_TO_STANDARD_BY_COL,
+    TEST_PCT_SHARE_MAP,
+    PCT_SHARE_MAP,
+    PCT_RELATIVE_INEQUITY_MAP,
+    BW_FLOAT_COLS_RENAME_MAP,
+    get_bq_col_types,
+)
+from ingestion.dataset_utils import (
+    generate_pct_share_col_without_unknowns,
+    generate_pct_rel_inequity_col,
+    preserve_only_current_time_period_rows,
+)
+from typing import cast
+
+CDC_AGE = "Age Group"

```

Comment:
these are already defined in the cdc hiv utils right? can we just import them like we are in the normal hiv file?